cav_msgs/msg/ManeuverPlan.msg

```diff
@@ -0,0 +1,13 @@
+# ManeuverPlan.msg
+#
+# Message defining a set of maneuvers which are meant to be executed in sequence
+#
+
+# Header
+std_msgs/Header header
+
+# The unique GUID of this maneuver plan
```

Comment:
Might be worth while to add a bit more metadata here, like a planning started/finished timestamp at least. The Header has a timestamp I believe, but that would normally be for message creation and might get changed as each plugin adds maneuvers to this plan.

Reply:
Makes sense I'll add that